### DISCUSSION TITLE
Remove control in card

### DISCUSSION THREAD

**Discussion by z1022** [Tasshack/dreame-vacuum#178] [Q&A]
<img width="331" alt="258596899-a047316f-4cbe-4a9b-b6d9-25fdc279f085" src="https://github.com/Tasshack/dreame-vacuum/assets/1883855/d4b17f0e-2dab-4ac9-b66e-60c8aecb99a9">
How to remove contro in cardl?  Thanks

**Comment by Tasshack** [comment]
Answer is not here, check https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card
